Assets/Tests/PlayMode/Sensors/SensorsTest.cs

```diff
@@ -7,11 +7,11 @@
 using UnityEngine.SceneManagement;
 using UnityEngine.TestTools.Utils;
 using AWSIM;
+using AWSIM.Tests;
 
 public class SensorsTest
```

Comment:
@szylis
Could you add a comment explaining the content of this test?

Reply:
@mackierx111 Done